packages/react/src/components/header/components/headerActionBar/HeaderActionBarNavigationMenu.tsx

```diff
@@ -415,20 +418,6 @@ export const HeaderActionBarNavigationMenu = ({
     setMobileMenuOpen(false);
   };
 
-  const setCurrentMenuHeight = () => {
```

Comment:
Just curious to know why it was possible to remove this? Because of the css change?

This was added to to force items in the mobile version to stay at the bottom when menus were navigated and animated.

The menu seems to work, so excellent change! 😄

Reply:
I fixed the css (It wasn't easy), but basicly I fixed flex-boxes to work correctly.